content/en/account_management/audit_trail/events.md

```diff
@@ -25,6 +25,7 @@ further_reading:
 - [Organization management](#organization-management-events)
 
 #### Product-Specific Events
+- [Application Security Management](#application-security-management)
```

Comment:
```suggestion
- [Application Security Management (ASM)](#application-security-management)
```
adding acronym on first reference, since it's used later